tests/locallib_test.php

```diff
@@ -29,15 +29,92 @@
 require_once($CFG->dirroot . '/mod/accredible/locallib.php');
 
 class mod_accredible_locallib_testcase extends advanced_testcase {
-    /**
-     * Sample test
-     * later: remove after adding real tests
-     */
-    public function test_sample() {
+    function setUp(): void {
         $this->resetAfterTest();
-        $this->setAdminUser();
-    
-        // Sample test.
-        $this->assertEmpty("");
+        $this->user = $this->getDataGenerator()->create_user();
+        $this->course = $this->getDataGenerator()->create_course();
+    }
+
+    public function test_accredible_get_transcript() {
+        global $DB;
+
+        /**
+        * When no quiz available for user.
+        */
+        $result = accredible_get_transcript($this->course->id, $this->user->id, 5);
```

Comment:
The third argument `$final_quiz_id` will be zero in the possible scenario when the course has no quizzes.